typescript/src/cli.ts

```diff
@@ -14,15 +14,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as yargs from 'yargs';
-import { Generator } from './generator';
+import * as yargs from "yargs";
```

Comment:
Gah, that isn't right :(  Can you copy the integrated prettier config from other repos into the root?  I think for the next lib, we should do `npx gts init` to let it bootstrap these files.

Reply:
Done! Looks much less scary now!